src/dialog/tooltip/tooltip.directive.ts

```diff
@@ -89,6 +92,6 @@ export class TooltipDirective extends DialogDirective {
 		this.dialogConfig.compID = "tooltip-" + TooltipDirective.tooltipCounter;
 		this.dialogConfig.content = this.ibmTooltip;
 		this.dialogConfig.type = this.tooltipType !== undefined ? this.tooltipType : this.type;
-		this.elementRef.nativeElement.setAttribute("aria-describedby", this.dialogConfig.compID);
+		this.descriptorID = this.dialogConfig.compID;
```

Comment:
```suggestion
		this.descriptorId = this.dialogConfig.compID;
```
to go along with the name change.